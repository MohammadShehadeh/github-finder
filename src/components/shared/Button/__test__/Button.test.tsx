import renderer from 'react-test-renderer';

import { Button } from '..';

describe('Button', () => {
  it('should render "default" to match snapshot', () => {
    const tree = renderer.create(<Button>Default Button</Button>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render "small" to match snapshot', () => {
    const tree = renderer
      .create(
        <Button variant="contained" size="small">
          small Button
        </Button>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
