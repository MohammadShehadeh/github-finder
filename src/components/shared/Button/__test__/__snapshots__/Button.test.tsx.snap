// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render "default" to match snapshot 1`] = `
<button
  className="button contained default"
>
  Default Button
</button>
`;

exports[`Button should render "small" to match snapshot 1`] = `
<button
  className="button contained small"
>
  small Button
</button>
`;
